wwwapp/migrations/0081_campparticipant_cover_letter.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.11 on 2022-02-02 18:57
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def forwards_func(apps, schema_editor):
+    UserProfile = apps.get_model("wwwapp", "UserProfile")
+    Camp = apps.get_model("wwwapp", "Camp")
+    for item in UserProfile.objects.all():
+        if not item.cover_letter:
+            continue
+
+        camp_participation = item.camp_participation.order_by('-year__year').first()
+        if not camp_participation:
+            camp_participation = item.camp_participation.create(year=Camp.objects.latest())
```

Comment:
I'm having second thoughts about this. If there is any user in the database who filled out the cover letter but never registered for any workshops, they will be added to the current edition (and receive emails intended for current interested participants). @cytadela8 could you check in django shell if this ever happens in the production db?